assets/sass/_mixins.scss

```diff
@@ -469,6 +469,31 @@ $highlight-color: $color-primary, $base-color: rgba(255, 255, 255, 0.2)) {
   vertical-align: middle;
 }
 
+@mixin background-icon($url, $size: 21px, $position-x: 50px, $position-y: 0, $width: 70px) {
+  display: none;
+
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+    @include margin(15, "right");
+    background-image: url($url);
+    background-repeat: no-repeat;
```

Comment:
background-image and background-repeat can be reduced into one line

Reply:
I made the change